src/extension/background-script/connectors/lndhub.ts

```diff
@@ -348,7 +353,12 @@ export default class LndHub implements Connector {
       data = res.data;
     } catch (e) {
       console.log(e);
-      if (e instanceof Error) throw new Error(e.message);
+      if (axios.isAxiosError(e)) {
+        const err = e as AxiosError;
+        const errResponse = err.response as AxiosResponse;
+        const errorMessage = `${errResponse?.data.message}\n(${e.message})`;
```

Comment:
I sneaked in some better error messages here

Reply:
@bumi I believe those type assertions as AxiosError and as AxiosReponse are redundant as TS understands that already because of the isAxiosError check?

Reply:
I had errors with those and have thrown everything at it to get it compile 🙈
how should it be written?

Reply:
just:
`const errorMessage = `${e.response?.data.message}\n(${e.message})`;`